### PR TITLE
SONARCHKST-36 Upgrade to Checkstyle 6.12

### PR DESCRIPTION
**Pull request by m-g-sonar**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -49,174 +61,8 @@
...
-          <threadCount>3</threadCount>
-          <perCoreThreadCount>true</perCoreThreadCount>
-        </configuration>
-      </plugin>
-      -->
-    </plugins>
-  </build>
-
-</project>
+</project>
```
Comment (mpaladin) [inline pom.xml]: @m-g-sonar EOF new line.

File: sonar-checkstyle-plugin/src/main/resources/org/sonar/l10n/checkstyle/rules/checkstyle/com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<p>Controls the indentation between comments and surrounding code. Comments are indented at the same level as the surrounding code. Detailed info about such convention can be found <a href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s4.8.6.1-block-comment-style">here</a></p>
```
Comment (mpaladin) [inline sonar-checkstyle-plugin/src/main/resources/org/sonar/l10n/checkstyle/rules/checkstyle/com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck.html]: @m-g-sonar EOF new line.

File: src/main/resources/org/sonar/l10n/checkstyle/rules/checkstyle/com.puppycrawl.tools.checkstyle.checks.blocks.EmptyCatchBlockCheck.html [hunk 3]
```diff
@@ -1 +0,0 @@
-Checks for empty catch blocks.
```
Comment (mpaladin) [inline src/main/resources/org/sonar/l10n/checkstyle/rules/checkstyle/com.puppycrawl.tools.checkstyle.checks.blocks.EmptyCatchBlockCheck.html]: @m-g-sonar EOF new line.

Comment (m-g-sonar) [inline src/main/resources/org/sonar/l10n/checkstyle/rules/checkstyle/com.puppycrawl.tools.checkstyle.checks.blocks.EmptyCatchBlockCheck.html]: Come on @mpaladin, I removed the file! :)

**Pull request merged**